### ISSUE TITLE
[D8] Theme sign-in form.

### ISSUE THREAD

**Issue by fureigh** [18F/web-design-standards-drupal-d7#43]
Stub.

**Notes:**
- Check whether #36 addresses this sufficiently.
- See #15 for complete list of components.
- See #42 for corresponding D7 issue.

**Comment by brockfanning** [comment]
I don't think this required any additional theming, beyond what USWDS does out of the box, but in #67 I did remove some unnecessary help text, and added the ability to log in using email or username (per the [guidelines](https://standards.usa.gov/components/form-templates/#sign-in-form).
